### PR TITLE
udp example

### PR DESCRIPTION
**Pull request by Pymossy**
create a basic example to use node-dvbtee with udp socket

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# dvbtee-express
+This is a basic example of how to use node-dvbtee with udp socket.
```
Comment (mkrufky) [inline README.md]: Please keep in mind, this `README.md` is for the entire repository.  Can you change "with udp socket" to something more like "using various interfaces / methods" or something else that you think may be more appropriate.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Thanks a lot to @mirmilad
```
Comment (mkrufky) [inline README.md]: not going to thank me ?  O:-)  I think we can omit this line from the `README.md` ... If you want to instead create a `CONTRIBUTORS` file, include mirmilad, yourself and I.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Extract EPG information and other metadata from mpegts udp stream.
```
Comment (mkrufky) [inline README.md]: Please either make this general or include usage info for all of the interfaces currently supported.

**Review by mkrufky — CHANGES_REQUESTED** [review]
Thanks for your contribution!  The only issues I have are with the `README.md` - remember that this repo is a basic example for various simple interfaces, not only udp.  If you want to have a `README.md` that's dedicated to udp, then it shouldn't be in the top level of the repository.  I'll be happy to merge once we can address that issue.

**Comment by Pymossy** [comment]
Thank you, I solved all issues

**Comment by mkrufky** [comment]
@Pymossy , pardon the late reply.  It's been a busy summer, and github notifications don't seem to be working for me :-/  I only noticed your comment yesterday.

I want to rewrite the README.md, so I'm going to rename yours to README-UDP for now, but I've merged your actual code changes in bcba8234027db19557917ae64a8b25f594a01fc9 to my master branch.

Again, thank you for your contribution.

**Pull request closed without merging**